docs/network-flow-visibility.md

```diff
@@ -1,12 +1,97 @@
-# ELK Flow Collector
-## Purpose
-Antrea supports sending IPFIX flow records through a flow exporter. The Elastic
+# Network Flow Visibility in Antrea
+
+## Table of Contents
+- [Overview](#overview)
+- [Flow Exporter feature](#flow-exporter-feature)
+  * [Configuration](#configuration)
+  * [Supported capabilities](#supported-capabilities)
+- [ELK Flow Collector](#elk-flow-collector)
+  * [Purpose](#purpose)
+  * [About Elastic Stack](#about-elastic-stack)
+  * [Deployment Steps](#deployment-steps)
+  * [Pre-built Dashboards](#pre-built-dashboards)
+    + [Overview](#overview-1)
+    + [Flows](#flows)
+      - [Pod-to-Pod Traffic](#pod-to-pod-traffic)
+      - [Pod-to-Service Traffic](#pod-to-service-traffic)
+    + [Flow Records](#flow-records)
+    + [Node Throughput](#node-throughput)
+      
+## Overview
+[Antrea](architecture.md) is a Kubernetes network plugin that provides network connectivity and security features for Pod workloads.
+Considering the scale and dynamism of Kubernetes workloads in a cluster, Network Flow Visibility helps in the management
+and configuration of Kubernetes resources such as Network Policy, Services, Pods etc., and thereby provides opportunities
+to enhance the performance and security aspects of Pod workloads.
+
+For visualizing the network flows, Antrea monitors the flows in Linux conntrack module. These flows are converted to flow records
+and are sent to the configured flow controller. High-level design is given below:
+
+<img src="https://s3-us-west-2.amazonaws.com/downloads.antrea.io/static/netviz.png" width="600" alt="Network Flow Visibilty">
```

Comment:
I think we should consider building a "cleaner" svg diagram in the future, that we can potentially use for slide presentations. But this is good enough got now.

Reply:
Yes, will keep a note of that.